hydra-node/test/Hydra/Chain/Direct/Contract/CollectCom.hs

```diff
@@ -145,6 +168,13 @@ genCollectComMutation (tx, utxo) =
             [ ChangeHeadDatum $ Head.Initial c moreParties
             , ChangeOutput 0 $ mutatedPartiesHeadTxOut moreParties
             ]
+    , SomeMutation MutateHeadId <$> do
+        illedHeadResolvedInput <-
```

Comment:
:joy: is that even a word? Maybe `sickHeadResolvedInput` is also an option?

No need to change, though.

Reply:
I thought that as a German-speaker, you would appreciate our creativity on words :nerd_face: